integreat_cms/cms/views/push_notifications/push_notification_form_view.py

```diff
@@ -96,17 +93,17 @@ def get(self, request, *args, **kwargs):
         push_notification_form = PushNotificationForm(instance=push_notification)
 
         num_languages = len(region.active_languages)
-        PNTFormset = modelformset_factory(
-            PushNotificationTranslation,
+        PushNotificationTranslationFormSet = inlineformset_factory(
```

Comment:
I suggest to use the `inlineformset_factory` instead of the `modelformset_factory` because the [BaseInlineFormSet](https://docs.djangoproject.com/en/3.2/topics/forms/modelforms/#django.forms.models.BaseInlineFormSet) is exactly designed for our purpose - saving multiple objects with a foreign key to the same object instance. Also, since the formset returns a class, I guess it's justified to write the variable name upper case.